File: 2026/05-May/27-Wednesday/solution.cs
public class Solution {
    public int NumberOfSpecialChars(string word) {

        HashSet<char> lower = new();
        HashSet<char> upper = new();

        foreach(char ch in word){
            if(char.IsLower(ch))
                lower.Add(ch);
            else
                upper.Add(char.ToLower(ch));
        }

        lower.IntersectWith(upper);

        return lower.Count;
    }
}

File: 2026/05-May/27-Wednesday/solution.go
func numberOfSpecialChars(word string) int {
    lower := map[rune]bool{}
    upper := map[rune]bool{}

    for _, ch := range word {
        if unicode.IsLower(ch) {
            lower[ch] = true
        } else {
            upper[unicode.ToLower(ch)] = true
        }
    }

    count := 0
    for ch := range lower {
        if upper[ch] {
            count++
        }
    }

    return count
}
